tests/test_files/html/tables.html

```diff
@@ -0,0 +1,4 @@
+<p>Colons can be used to align columns.</p>
```

Comment:
Do you want text in here indicating that tables will be stripped from html?

Reply:
Not here.  I documented that in the index file.  The input markdown reflected here is copied directly from [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables).

Reply:
I just mean to write some text in here like you did for the images.html that says the table is going to be stripped. I found it confusing to read the text that is here and to not see any tables, but I will defer to your judgement.